src/SurveyService.Data/OptionRepository.cs

```diff
@@ -0,0 +1,31 @@
+﻿using LT.DigitalOffice.SurveyService.Data.Interfaces;
+using LT.DigitalOffice.SurveyService.Data.Provider;
+using LT.DigitalOffice.SurveyService.Models.Db;
+using System;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.SurveyService.Data;
+
+public class OptionRepository : IOptionRepository
+{
+  private readonly IDataProvider _provider;
+
+  public OptionRepository(
+    IDataProvider provider)
+  {
+    _provider = provider;
+  }
+
+  public async Task<Guid> CreateAsync(DbOption dbOption)
+  {
+    if (dbOption == null)
+    {
+      return default;
```

Comment:
return null
what if dbOption is null, you return 'default', in Command there is no check for default, and Guid=default goes to response - it's not good
better return 'null' instead 'default' and add null-check to command

Reply:
Done